pkg/utils/parse_arf_result.go

```diff
@@ -135,6 +136,39 @@ func warningHasApiObjects(in *xmlquery.Node) bool {
 	return false
 }
 
+// compare resourcePath arrays
+func CompareResourcePaths(a, b []ResourcePath) bool {
```

Comment:
Since these functions are only used in tests, can we move them outside CO code and into the tests?